From af6ef89d6f23f6f4470ee502748c35f9f094a700 Mon Sep 17 00:00:00 2001
From: Joseph Chen <chenjh@rock-chips.com>
Date: Mon, 28 Jan 2019 11:24:31 +0800
Subject: [PATCH] arm64: dts: rockchip: set pwm regulator in default voltage
 for rk3308k

rk3308k supports wide temperature feature, it makes system suspend
stable in extrem low temperature.

Change-Id: I07427c21263e5a48bc07c935291f8494e50ec9e3
Signed-off-by: Joseph Chen <chenjh@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3308k.dtsi | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3308k.dtsi b/arch/arm64/boot/dts/rockchip/rk3308k.dtsi
index 0a7a6396537e..9b41bcb4f2eb 100644
--- a/arch/arm64/boot/dts/rockchip/rk3308k.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3308k.dtsi
@@ -13,6 +13,16 @@
 	};
 };
 
+&rockchip_suspend {
+	rockchip,sleep-mode-config = <
+		(0
+		| RKPM_PMU_HW_PLLS_PD
+		| RKPM_DBG_FSM_SOUT
+		| RKPM_PWM_VOLTAGE_DEFAULT
+		)
+	>;
+};
+
 &thermal_zones {
 	soc-thermal {
 		sustainable-power = <308>;
-- 
2.35.3

